bitnami/fluentd/values.yaml

```diff
@@ -970,7 +970,9 @@ aggregator:
     ##                 operator: In
     ##                 values:
     ##                   - frontend
-    extraIngress: []
+    extraIngress:
+       - ports:
+           - port: 24224
```

Comment:
Hello @opencmit2! I agree with the fix but I think we should add this to the NetworkPolicy template instead of having the `extraIngress` filled by default. I'll say we could add the following to its [template](https://github.com/bitnami/charts/blob/a718e62906a620b2520e1c80a98ffa040505cce6/bitnami/fluentd/templates/aggregator-networkpolicy.yaml#L65-L66):

```diff
        {{- range .Values.aggregator.containerPorts }}
        - port: {{ .containerPort }}
        {{- end }}
+       {{- if .Values.aggregator.port }}
+       - port: {{ .Values.aggregator.port }}
+       {{- end }}
        {{- if .Values.metrics.enabled }}
```

Reply:
@FraPazGal 
Revisions have been made. Please review.